### ISSUE TITLE
Exiting Multi-Line Cursor Doesn't Return to the Original Cursor

### ISSUE THREAD

**Issue by Emmennater** [processing/p5.js-web-editor#3301]
### p5.js version

1.11.1

### Web browser and version

Chrome 131.0.6778.86 (Official Build) (64-bit) (cohort: Stable)

### Operating system

Windows 10

### Steps:
1. Place your cursor somewhere in the file.
2. Make a multiline cursor any way you like (ctrl+alt+down/up arrow, highlight and ctrl+d)
3. Press escape. (this should return to your original cursor when you first clicked)

# Initial click
![image](https://github.com/user-attachments/assets/854c38a2-e311-4461-b480-a9a2c4896a0c)
# Multiline cursor down (ctrl+alt+downarrow)
![image](https://github.com/user-attachments/assets/ada5d2a6-5ad2-4b63-8439-25a83185c5ea)
# After pressing escape
![image](https://github.com/user-attachments/assets/93e48d59-2094-45a3-bbe7-0d8b2f814ea6)
# Expected behavior
![image](https://github.com/user-attachments/assets/e217e5eb-22c2-4cbd-a68c-e7365ee3caac)

**Comment by Emmennater** [comment]
For who ever sees this, this should probably be under p5js-web-editor. I am not sure how to move the issue or if I have permission.

**Comment by davepagurek** [comment]
Right, that's probably the right home for it, maybe the thing to do is close this issue, and open a new one over there, and just copy and paste the contents of this issue into your new one?

**Comment by raclim** [comment]
Thanks for raising this issue and forwarding it this way! I edited the original post a little to help reformat!
